tests/pc/windows/visual_studio/aws_tests.vcxproj.filters

```diff
@@ -561,9 +564,6 @@
     <ClInclude Include="..\..\..\..\lib\include\aws_tls.h">
       <Filter>lib\aws\include</Filter>
     </ClInclude>
-    <ClInclude Include="..\..\..\..\lib\include\aws_ota_agent.h">
```

Comment:
Is this expected?

Reply:
I suppose so. I didn't touch it. But it was automatically changed to:

<ClInclude Include="..\..\..\..\lib\include\aws_iot_ota_agent.h" />

Reply:
After merged latest code, this problem seems to be auto-resolved.